src/routes/signup.tsx

```diff
@@ -90,16 +103,25 @@ const SignUpForm = () => {
             minLength={minPasswordLength}
             maxLength={maxPasswordLength}
           />
-          <Dropdown<Realm>
-            value={realms.find((r) => r.id === realmId)}
-            emptyLabel="Select a realm"
-            class={dropdownClass}
-            options={realms}
-            required
-            onChange={(v) => setRealmId(v.id)}
-            getKey={(v) => v.id}
-            getText={(v) => v.name}
-          />
+          <div class={realmElementsStyle}>
+            <Dropdown<Realm>
+              value={realms.find((r) => r.id === realmId)}
+              emptyLabel="Select a realm"
+              class={dropdownClass}
+              options={realms}
+              required
+              onChange={(v) => setRealmId(v.id)}
+              getKey={(v) => v.id}
+              getText={(v) => v.name}
+            />
+            <div class={textStyle}>
+              Don&apos;t see your team? Ask a captain or mentor to make a new
+              realm.{' '}
```

Comment:
```suggestion
              {"Don't see your team? Ask a captain or mentor to make a new realm."}
```
Prettier might format this a little different. Not sure if it will change it to multiple lines.

Reply:
Actually, if I remember right, you might not even need the curly brackets and quotes. I think you can just put an apostrophe in and it will convert to JS correctly and the HTML escaping will happen automatically